src/libServer/Server.cpp

```diff
@@ -192,7 +192,7 @@ Json::Value Server::GetDsBlock(const string& blockNum)
 
     try
     {
-        boost::multiprecision::uint256_t BlockNum(blockNum);
+        uint64_t BlockNum = stoi(blockNum);
```

Comment:
Using `stoi` will bring two possible exceptions. Since we've already had multiple `catch` underneath, can we also add some handling and logging for these new exceptions?

About `stoi`'s exceptions: http://en.cppreference.com/w/cpp/string/basic_string/stol#stoi

Reply:
should catch invalid argument